Hai Reveal intro: rewrite pug media coverage with less narrator editorializing, as several missions in a new file (#7760)
endless-sky/endless-sky: data/human/post-war reactions.txt

				goto next
			label blew
			`	"I bet you did! I wish I was on one of our warships, bravely defending us all. Blowing up Pugs, getting paid. It sounds amazing!"`
			label next
			`	"What was it like, fighting those nasty, evil Pugs?"`

			choice
				`	"Actually, they're not evil. Their 'war' was a ploy to get us to stop fighting among ourselves. A lot of Pug gave their lives willingly to save us from self-destruction."`
				`	"War is hell, kid. I'm not going to glorify it for you."`
					goto hell
				`	"Leading a battle is like chess with guns and engines. It's the most exhilarating feeling I've ever felt."`
					goto chess
				`	"It was an honor to protect the human race from our attackers."`
					goto honor
			`	"Woah, you sound really smart. Like one of those sensible experts in the vids! Still, without people like you, the Pug's plan couldn't have worked. I still reckon you were the best, Captain."`
				goto salute
			label hell
			`	"You went through hell for us, eh? That sounds less fun than I thought... Maybe I should stick to school and be an engineer instead of a pilot. I dunno."`
				goto salute
			label chess

> 			`	"The Syndicate is training me to maintain and they're even going to train me to pilot their combat ships when I'm older! One day, I'll fight aliens like you did."`

Reviewer: ```suggestion
			`	"The Syndicate is training me to maintain their combat ships, and they're even going to train me to pilot them when I'm older! One day, I'll fight aliens like you did."`
```

I know it loses hurting the lead, but I honestly can't think of a way to make this less awkward than shifting that back.

Dropped comma needed either way :)

Reviewer 2: It's structured for the manner of a 12-year old. It's *meant* to sound awkward to an adult.

				goto salute
			label honor

> 			`	"Honor, and service. The Syndicate will train boys like me, but I've wondered about joining the Navy. It isn't as profitable, but it sounds more noble."`

Reviewer: ```suggestion
			`	"Honor... and service. The Syndicate will train boys like me, but I've wondered about joining the Navy. It isn't as profitable, but it sounds more noble."`
```

If the point is to have a pause in the kid's speech here, I think this route will be better.

If it's not, just cut the extra comma.

			label salute
			`	You arrive at the repair shop, and an older girl, perhaps seventeen, meets the two of you. Gus stands to attention and gives you a well-practiced salute. You politely salute him in return.`
			`	As Gus ends his salute, the girl attacks him from behind, putting him into a choke hold. It's clear she's taking extreme care when doing it. "Did you wander out of the childcare district? Was it the spaceport landing area this time? I can see scratches and bruises all over you." Gus replies, "I'm sorry, Sally! I was curious."`
			`	The girl lets him out of the hold and says to you, "Thank you for saving my brother. He's the only family I have left, and I hope," she pinches him, "he'll stop risking his life out of curiosity." She looks at Gus kindly. "Come on, let's get you to the medic."`

> 			`	Sally waves goodbye as she walks her brother into the repair shop orphanage.`

Reviewer 2: I think this could optionally just simply be truncated with the new context.
```suggestion
			`	Sally waves goodbye as she walks her brother in.`
```

# Copyright (c) 2022-2023 by an anonymous author
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE. See the GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License along with
# this program. If not, see <https://www.gnu.org/licenses/>.

########################################################################

# This file contains human reactions to the pug war, and events
# leading up to it.  It lets us know how regular, everyday people felt
# about the war. This leads to the reveal of the Hai, who are unable
# to stay hidden with so many people thinking about aliens.

> mission "00 Post-War Reactions Begin"

Author: I need this mission to trigger before several others. Missions trigger in order of ASCII lexical sorting. The "00" puts it before most missions.

> mission "00 Post-War Reactions Begin"

Reviewer 3: The fact that it's non-minor and there are no priority missions, at least that I see, and this mission contains no conversation or dialog, means that there's no need to manipulate the offer order of this mission.

> mission "00 Post-War Reactions Begin"

Author: My memory is hazy of when I tested this in Hai Reveal, so I'll take your word for it and remove the 00.

> mission "00 Post-War Reactions Begin"

Author: The 00 is gone.

	landing
	invisible
	to offer
		has "main plot completed"
	on offer
		event "post-war reactions begin" 20 30
		fail

event "post-war reactions begin"

mission "War Reactions: Asteroid Miners"
	landing
	minor
	source
		government "Republic" "Syndicate" "Free Worlds"
		# These miners were far from the war the whole time.
		attributes "deep" "paradise" "north"
		not attributes uninhabited
	to offer
		has "event: post-war reactions begin"
		random < 30
	on offer

> 		set "pug media coverage"

Author: The variable is misnamed, and I kept it that way to avoid breaking old saves. It should be "awareness of alien activity" since it indicates the player knows that human awareness is increasing. The reason for the incorrect name, "pug media coverage," is that it is replacing a mission that just showed pug media coverage.

If you have suggestions on how to rename this without breaking old saves, I'm open to them.

> 		set "pug media coverage"

Author: Also, these changes, plot-wise, should be entirely separate from Hai Reveal, and aren't specific to the Pug War. They show how regular people were affected by the events of the main campaign.

> 		set "pug media coverage"

Reviewer 3: HR has only ever been in continuous, and so we should not be keeping improper names for the sake of save compatibility. Part of playing on continuous is the expectation that things might break.

> 		set "pug media coverage"

Author: Okay, I just removed the "Fix for pug media coverage" mission. There is now no fix for broken saves.

			choice
				`	"It was possibly the biggest event this century. Of course people are going to keep talking about it."`
				`	"Don't you see it was what we needed? The Pug made us set aside our pride and resolve our issues directly without continuing the conflict."`
				`	"Oh, the Pug were real alright. I met some and fought them."`
					goto nobody
				`	"I think you both need to shave."`
					goto scowl
			`	One twin rolls his eyes while the other twitches his mustache awkwardly. He rewinds the video to the Parliamentary discussion, where Danforth is reassuring councilors about the Navy's ability to patrol the fringes of human space.`
			`	The left twin says, "Look at this bozo! Bounty's Councilor Maduena Kabiru, he keeps ranting against this Navy guy, uh, this Admiral..." He snaps his fingers trying to remember the name.`
			`	"Admiral Danforth," says the right twin.`
			`	"Right. Danforth. This Councilor wants Danforth to defend against 'unknown dangers at the fringes of human space'. Don't we have enough immediate problems without imagining new horrors?"`

> 			`	"Like terrorists nuking us," says the right twin.`

Author: The right twin is emphasizing the left twin's point. "Terrorists nuking us" refers to a past event that would have seemed impossible a few years ago. The point is that things are already pretty bad, so we don't need to invite more trouble.

			choice
				`	"Have you been under a rock the whole war? I'm Captain <first> <last>!"`
				`	"Never mind."`
					goto shrug
				`	"At least I don't have a disheveled hamster stuck to my upper lip."`
					goto scowl
			`	Both twins look confused, and the left one responds. "We've been mining asteroids the whole war. The war was a big deal, but most of us had to keep living our lives. Do you think we have time to keep track of every warring captain?"`
			`	The right one says apologetically, "Thank you for your service, Captain <last>."`
			`	They both give clumsy but well-intentioned salutes and return to their ships.`
				decline
			label eyes
			`	The left twin laughs and says, "Pardon us if we keep our eyes open and trust to our own guns." Both twins nod and depart to their ships.`
				decline
			label shrug
			`	Both twins shrug and leave for their ships.`
				decline
			label scowl
			`	Both twins' faces twist in a way that might indicate embarrassment or annoyance. It's hard to tell since their facial hair is halting any emotive power. They walk away in the direction of their ships.`
				decline

mission "War Reactions: Navy and Free Worlds Captains"

> 	landing

Reviewer 4: `You are relaxing in a cafe [...]` doesn't really sound like the beginning of a `landing` mission.
```suggestion
```

Author: No, it doesn't sound that way to me either, but there are a lot of Hai Reveal landing missions that are written like spaceport missions. I'll apply your change, but I'll leave this unresolved for further discussion.

	minor
	source
		government "Republic" "Syndicate" "Free Worlds"
		attributes "spaceport"
		# This distance from Sirius puts them a few jumps from the edge of human space, or anywhere in the Free Worlds

> 		near "Sirius" 5 100

Reviewer 4: It also seems to include much of the Deep. While those systems are near the edge of human space, it's not really in the same way as the systems, for example, in the far north are.

Testing this filter with relevant changes (this also includes all the changes from HR and the Wanderer story because I can't be bothered finding a save file with only the changes from the FW story right now):

<details> <summary>Planets</summary>

- Albatross
- Alexandria
- Alfheim
- Amazon
- Antipode
- Arabia
- Asgard
- Big Sky
- Bivrost
- Bloodsea
- Bluestone
- Bounty
- Bourne
- Burthen
- Calda
- Canyon
- Charon Station
- Clark
- Cornucopia
- Crossroads
- Dancer
- Darkstone
- Deep
- Dune
- Farpoint
- Featherweight
- Fenrir Station
- Flood
- Foundry
- Gemstone
- Glaze
- Glory
- Harmony
- Haze
- Heartland
- Helheim
- Hippocrates
- Hopper
- Icefall
- Ingot
- Leviathan Station
- Lichen
- Longjump
- Maelstrom
- Mani
- Memory
- Mere
- Midgard
- Moonshake
- Muninn Station
- Muspel
- New Argentina
- New Britain
- New Greenland
- New Holland
- New Iceland
- New India
- New Kansas
- New Portland
- New Tibet
- New Wales
- New Washington
- Nifel
- Nimbus
- Norn
- Oasis
- Oblivion
- Pearl
- Placer
- Poisonwood
- Poseidos
- Prime
- Quicksilver
- Rand
- Rust
- Serpens
- Shangri-La
- Shorebreak
- Shroud
- Sinter
- Skillet
- Skymoot
- Solace
- Splashdown
- Starcross
- Sundive
- Sundrinker
- Sunracer
- Thunder
- Trinket
- Triton Station
- Trove
- Twinstar
- Vail
- Valhalla
- Wayfarer
- Windblain
- Winter
- Wyvern Station
- Zug

</details>

Author: The Deep is intentionally part of the filter. I'm selecting systems that have not been touched by the main campaign conflict. Quicksilver should not have been on that list, so I do need to update it.

Author: This save has done the main campaign without ever leaving human space: 
[False Facsimile~3018-12-23 main campaign done.txt](https://github.com/endless-sky/endless-sky/files/11244725/False.Facsimile.3018-12-23.main.campaign.done.txt)

	to offer
		has "event: post-war reactions begin"
		random < 30
	on offer

> 		set "pug media coverage"

Reviewer 4: ```suggestion
		"pug media coverage" ++
```

			`	Rain continues, "You do what you have to to follow your orders and look out for each other in between."`
				goto questions
			label mercenary
			action
				set "War Reactions: Navy and Free Worlds Captains: label mercenary"
			`	"I meant no offense," says Fiona, "you're quite the hero." She thinks for a second and glances at her sister. "Rain, can you explain?"`
			`	Rain says, "We're uniformed career officers who must follow orders. Our states own the ships, uniforms, and everything else we use. Our salaries may be meager compared to your earnings, but serving one's state is a rich reward.`
			`	"You, on the other hand, own your ship and serve whoever you want. Even if you remain loyal to a group or cause, how and when you act is still at your own discretion. As well, to keep up with the costs of that independent ownership, you have to charge a lot of money for your services. Mercenaries only survive as long as you if they're skilled at both finances and combat."`
				goto questions

mission "War Reactions: Syndicate Orphan"
	landing
	minor
	source
		government "Syndicate"
		not attributes "station"
	to offer
		has "event: post-war reactions begin"
		random < 30
	on offer

> 		set "pug media coverage"

Reviewer 4: ```suggestion
		"pug media coverage" ++
```

			`	"Wait a minute!" He suddenly looks at you with wide eyes. "You're a captain, right? Were you in the big fights? I was on Maker when I saw some Pug ships fighting our ships on our school's big edu-screen."`
			choice
				`	"Yes, I'm Captain <first> <last>."`
					goto me
				`	"I blew up my share of Pug ships."`
					goto blew
				`	"Actually, I led the battle."`
			`	"Oh, hold on," he says, pulling out a durable-looking data pad labeled 'Syndicate Childcare Device'. After some typing and swiping, he declares, "You're Captain <first> <last>! Wow! It's an honor to meet you, sir. Can I shake your hand?" He puts out his hand, and you shake it.`
				goto next
			label me
			`	"Oh, hold on," he says, pulling out a durable-looking data pad labeled 'Syndicate Childcare Device'. After some typing and swiping, he declares, "You led the attack! Wow! It's an honor to meet you, sir. Can I shake your hand?" He puts out his hand, and you shake it.`
				goto next
			label blew

> 			`	"I bet you did! I wish I was on one of our warships, bravely defending us all. Blowing up Pugs, getting paid. It sounds amazing!"`

Reviewer 4: Yeah, just don't hope to be on my flagship: boarding Pug ships is dangerous, especially for children.

Author: He is too young to have considered those consequences, and he likely isn't aware of the Pug fondness for bioweapons.
